wgpu/examples/water/main.rs

```diff
@@ -528,7 +528,10 @@ impl framework::Example for Example {
             // Fragment shader and output targets
             fragment: Some(wgpu::FragmentState {
                 module: &water_module,
+                #[cfg(not(feature = "webgl"))]
                 entry_point: "fs_main",
```

Comment:
if you can use run-time conditions, it's always preferred over compile time, since both branches are checked at compilation: `entry_point: if cfg!(something) { ... } else { ... },`

Reply:
Ah, good point. I hadn't thought of that before. I pushed an update.